### PR TITLE
docs: updated docs with .zenodo.json material

### PR DESCRIPTION
**Pull request by ctrl-alt-soneca**

### CODE REVIEW

**Comment by ctrl-alt-soneca** [comment]
Captura de ecrã 2021-06-28, às 12 54 32
Captura de ecrã 2021-06-28, às 12 54 39

File: source/includes/github/_root.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    "access_right": "open",
+    "related_identifiers": [
+        {
+            "scheme": "doi",
+            "identifier": "10.5072/zenodo.<RECID>",
+            "relation": "isDocumentedBy"
+        }
+    ],
+    "keywords": ["Cats", "Laser", "Behavior"],
+    "grants": [{"id":"777541"}]
```
Comment (slint) [inline source/includes/github/_root.md:45]:
```suggestion
    "creators": [
        {   
            "orcid": "0000-0002-1825-0097",
            "affiliation": "Feline reasearch institute",
            "name": "Field, Gar"
        },
        {   
            "orcid": "0000-0002-1825-0097",
            "affiliation": "Feline reasearch institute",
            "name": "Cat, Felix"
        }
    ],    

    "license": "Apache-2.0",
    
    "title": "Red-Dot: ML-powered laser vector detection",
    
    "related_identifiers": [
        {
            "scheme": "doi",
            "identifier": "10.1234/software.paper.5678",
            "relation": "isDocumentedBy",
            "resource_type": {
                "type": "publication",
                "subtype": "article"
            }
        }
    ],
    
    "keywords": ["Cats", "Laser", "Behavior"],

    "communities": [
        {"identifier": "software-cats"}
    ],
    
    "grants": [{"id":"777541"}]
```

File: source/includes/github/_root.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+}
+```
+
+## How to verify your .zenodo.json?
+Now that you have edited your ``.zenodo.json`` file you should validate it to make sure everything is following the norm. In order to validate it, you can use a tool like the [JSON Validator](https://jsonformatter.curiousconcept.com/#).
```
Comment (slint) [inline source/includes/github/_root.md:50]:
```suggestion
## How to verify your `.zenodo.json`?

After creating your `.zenodo.json` file you should validate it to make sure that it contains valid JSON. You can use a tool like the [JSON Formatter & Validator](https://jsonformatter.curiousconcept.com), or load it via your favorite programming language.
```

**Review by slint — COMMENTED** [review]
Some sentence and content comments, but looking good otherwise :)

File: source/includes/github/_root.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
+# Github
+
+## Add metadata to your GitHub repository release
+
+Zenodo automatically extracts metadata about your repository from GitHub APIs. For example, the authors are determined from the repository's contributor statistics. To achieve overwriting some of the default metadata that would come from a regular GitHub release you will have to use the ``.zenodo.json`` file at the root of your GitHub repository.
+
+The ``JSON Schema`` followed by the ``.zenodo.json`` file is at [JSON Schema](https://zenodo.org/schemas/deposits/records/legacyrecord.json) and is documented in our developer docs at [Developer Docs](https://developers.zenodo.org/?python#deposit-metadata). After the first release of your repository you can also find a personalized version of the ``.zenodo.json`` on ``https://zenodo.org/account/settings/github/repository/`` which might be an easier alternative to edit.
```
[the remark this answers is not in the document]
Comment (ctrl-alt-soneca) [reply source/includes/github/_root.md:7]: Makes sense! :)

**Pull request merged**